bemani/backend/iidx/factory.py

```diff
@@ -81,9 +83,9 @@ def version_from_date(date: int) -> Optional[int]:
                 return VersionConstants.IIDX_COPULA
             if date >= 2016102600 and date < 2017122100:
                 return VersionConstants.IIDX_SINOBUZ
-            if date >= 2017122100:
+            if date >= 2017122100 and date < 2019012300:
                 return VersionConstants.IIDX_CANNON_BALLERS
-            return None
+            return VersionConstants.IIDX_ROOTAGE # date >= 2019012300
```

Comment:
Prefer putting this in an if statement and continuing to return None at the bottom as a safeguard against future modifications accidentally defaulting to rootage.

Reply:
You're right, I havn't considered that much, thanks ;)